guides/common/modules/con_configuring-dns-dhcp-and-tftp.adoc

```diff
@@ -0,0 +1,8 @@
+[id="configuring-dns-dhcp-and-tftp_{context}"]
+= Configuring DNS, DHCP, and TFTP
+
+If you want to manage DNS, DHCP, and TFTP centrally within the {Project} environment, you can configure the services directly on the {ProjectServer}.
+You can also manage the services manually, which involves disabling their maintenance on {Project} and configuring them independently.
+ifndef::foreman-deb,orcharhino[]
+Or you can run the services externally, outside of the {Project} environment.
```

Comment:
```suggestion
You can run the services externally, outside of the {Project} environment.
```
Instead of beginning the sentence with "Or", you can say it like this or even as "You can also run the services externally, outside of the {Project} environment."

Reply:
I had to tweak the previous sentences to avoid using "You can" twice in a row. Thanks for the catch!